src/components/BudgetKeyFooterComponent.ts

```diff
@@ -1,14 +1,13 @@
 import {Component, Inject} from '@angular/core';
 import {THEME_TOKEN} from '../constants';
-const Smooch: any = require('smooch');
```

Comment:
You should rebase your branch to get the latest code - otherwise you're going to make unwanted changes to master.